recipe/meta.yaml

```diff
@@ -6,32 +6,66 @@ package:
   version: {{ version }}
```

Comment:
@SrNetoChan should this be version 5.12.2 ?

Reply:
I was trying to make it work for 5.9 because that was the latest webkit release, then I would move to a higher version. Do we need to call it 5.12.2 if we want to use it with QT5.12, even if based on webkit 5.21 alpha?

(Reminder, I am not a conda or building expert, I am just trying stuff out)

Reply:
I *think* we need to use `qtwebkit` 5.9 with `qt` 5.9 and `qtwebkit` 5.12 with `qt` 5.12 (what conda forge is currently using). So I guess we need to decide what we are targeting. The current `meta.yaml` seems to be using `qtwebkit` 5.9 for Unix and 5.12 for Windows.... Note there is an alpha 4 out for `qtwebkit` 5.12 now.

Reply:
@gillins thanks. In that case, I will close this PR and move to  #13. It seems that currently, the build instructions for 5.21alpha are a bit different than before. Using conan and python to build and no longer perl. In that PR I am trying to create the bld.bat almost from scratch using the build instructions, but my knowledge is way too low... So I am giving baby steps.

Reply:
:+1:  Baby steps sounds good....